### PR TITLE
ISPN-12158 Remove stuff that was deprecated < 11.0

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-12158

### CODE REVIEW

File: commons/all/src/main/java/org/infinispan/commons/util/SmallIntSet.java [hunk 1]
```diff
@@ -25,49 +25,47 @@
  *
  * @author Dan Berindei
  * @since 9.0
- * @deprecated since 9.3 This class will no longer be public. Please use {@link IntSets} methods such as
  * {@link IntSets#mutableEmptySet()}, {@link IntSets#mutableCopyFrom(Set)}
```
Comment (wburns) [inline commons/all/src/main/java/org/infinispan/commons/util/SmallIntSet.java:28]: This line can be removed as well

File: commons/all/src/main/java/org/infinispan/commons/util/ImmutableIntSet.java [hunk 2]
```diff
@@ -12,13 +12,11 @@
  *
  * @author Dan Berindei
  * @since 9.2
- * @deprecated since 9.3 This class will no longer be public, please use {@link IntSets#immutableSet(IntSet)}
  */
-@Deprecated
-public class ImmutableIntSet extends AbstractImmutableIntSet {
+class ImmutableIntSet extends AbstractImmutableIntSet {
```
Comment (wburns) [inline commons/all/src/main/java/org/infinispan/commons/util/ImmutableIntSet.java:16]: The new OffsetIntSet can also be made package private. It was just added in 15.0, so no worries about deprecation.

**Review by wburns — CHANGES_REQUESTED** [review]
There are a lot of failures :(

File: commons/all/src/main/java/org/infinispan/commons/util/SmallIntSet.java (hunk 1 above)
Comment (tristantarrant) [reply commons/all/src/main/java/org/infinispan/commons/util/SmallIntSet.java]: Done

File: commons/all/src/main/java/org/infinispan/commons/util/ImmutableIntSet.java (hunk 2 above)
Comment (tristantarrant) [reply commons/all/src/main/java/org/infinispan/commons/util/ImmutableIntSet.java:16]: Done

**Comment by tristantarrant** [comment]
@wburns this should be ready

**Comment by tristantarrant** [comment]
One failure :/

**Comment by tristantarrant** [comment]
This is now good to go

**Comment by ryanemerson** [comment]
Thanks @tristantarrant

**Pull request merged**